{% extends "base.html" %}
{% block title %}Lute error{% endblock %}
{% block header %}500 internal server error{% endblock %}
{% block body %}
    <p>
        <i>Hmm, something went wrong ...</i>
    </p>
    <div class="bug_report">
        <p>Platform: {{ platform }}</p>
        <p>Version: {{ version }}</p>
        <p>In docker?: {{ is_docker }}</p>
        <p>Stack trace:</p>
        <div class="code">
            <code><pre>{{ exception_info }}</pre></code>
        </div>
    </div>
    {% if "CERTIFICATE_VERIFY_FAILED" in exception_info %}
        <blockquote>
            <p>This looks like an SSL certificate verification error.</p>
            <p>
                Possible solutions are outlined in
                <a href="https://stackoverflow.com/questions/42098126/mac-osx-python-ssl-sslerror-ssl-certificate-verify-failed-certificate-verify"
                   target="_blank">this StackOverflow post</a> ... see if any of these are effective.
            </p>
            <p>After applying any solution, restart Lute.</p>
        </blockquote>
    {% endif %}
    <p>
        If this issue persists, please open a
        <a href="https://github.com/jzohrab/lute-v3/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=500%20error"
           target="_blank">GitHub Issue</a>, including all of the above information in the issue.
    </p>
{% endblock %}
